Fashionz/assets/styles.css

```diff
@@ -0,0 +1,201 @@
+/*Universal css to start at 0,0*/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
```

Comment:
1. Where are importing the font Roboto from ? I don't see a link in css or HTML, and iff not imported it would be sans-serif
2. More over while using font-weights in your app will potentially break if the sans-serif doesn't support them

Reply:
Review comments resolved